src/components/Header/components/WalletIcon/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core/styles'
+import * as React from 'react'
+
+import Col from 'src/components/layout/Col'
+import Img from 'src/components/layout/Img'
+import WALLET_ICONS from './icons'
+
+const useStyles = makeStyles({
+  container: {
+    marginLeft: '5px',
+    marginRight: '10px',
+    letterSpacing: '-0.5px',
+  },
+  icon: {
+    maxWidth: 'none',
+  },
+})
+
+interface WalletIconProps {
+  provider: string
+}
+
+const WalletIcon: React.FC<WalletIconProps> = ({ provider }) => {
+  const classes = useStyles()
+  return (
+    <Col className={classes.container} layout="column" start="sm">
+      <Img
+        alt={provider}
+        className={classes.icon}
+        height={WALLET_ICONS[provider].height}
+        src={WALLET_ICONS[provider].src}
+        srcSet={WALLET_ICONS[provider].srcSet}
```

Comment:
Why do we need a responsive image for 25x25 size? Also you're not describing when to show an image.
The correct usage of `srcSet` attribute is
```
<img srcset=" url size, url size, url size " src="default url" >
```

Read more: https://html.com/attributes/img-srcset